src/app/services/map-cell.service.ts

```diff
@@ -0,0 +1,22 @@
+import { Injectable } from '@angular/core';
+import { Response } from '@angular/http';
+import { Point } from 'geojson';
+import { Observable } from 'rxjs/Observable';
+
+import { MapCell } from '../models/map-cell.model';
+import { LabApiHttp } from '../auth/api-http.service';
+import { apiHost } from '../constants';
+
+@Injectable()
+export class MapCellService {
+
+    constructor(private apiHttp: LabApiHttp) {}
+
+    nearest(point: Point, distance: number): Observable<MapCell[]> {
+        const url = `${apiHost}/api/map-cell/${point.coordinates[1]}/${point.coordinates[0]}?distance=${distance}`;
+        return this.apiHttp.get(url)
+        .map((resp) => {
```

Comment:
Minor thing, but I think it might be slightly more readable to shift this whole block inward by one indent. I had a moment of surprise seeing the `.map` aligned with `return`.

Also, why did you choose to use `.map` here rather than `.then`? Is that the preferred style for Angular?

Reply:
> Minor thing, but I think it might be slightly more readable to shift this whole block inward by one indent. I had a moment of surprise seeing the `.map` aligned with `return`.

:+1: 

> Also, why did you choose to use `.map` here rather than `.then`? Is that the preferred style for Angular?

This isn't mapping over a `Promise`, this is using the `rxJS` API, which does not have a `then` method: https://rxjs-dev.firebaseapp.com/api